cms/articles/models.py

```diff
@@ -446,9 +445,39 @@ def figures_used_by_ancestor(self) -> list[str]:
         topic_page_class = resolve_model_string("topics.TopicPage")
         topic = topic_page_class.objects.ancestor_of(self).first().specific_deferred
         return [
-            figure.value["figure_id"] for figure in topic.headline_figures if figure.value["series"].id == series.id
+            figure.value["figure_id"]
+            for figure in topic.headline_figures
+            if figure.value["series"].id == series.id
+            and figure.value["figure_id"] in self.headline_figures_figure_ids_list
         ]
 
+    @property
+    def figures_in_use_with_no_fallback(self) -> set[str]:
```

Comment:
Clearer as `figures_used_by_ancestor_with_no_fallback`?. Don't mind the naming as long as the 2 methods are consistent.

Reply:
Addressed in [245cc11](https://github.com/ONSdigital/dis-wagtail/pull/438/commits/245cc11162182024106f586c175def076a71ed9d).